src/sys/windows/tcp.rs

```diff
@@ -365,6 +365,7 @@ impl TcpListener {
 
     pub fn accept(&self) -> io::Result<(TcpStream, SocketAddr)> {
         try_io!(self, accept).and_then(|(inner, addr)| {
+            self.io_blocked_reregister()?;
```

Comment:
Maybe we should ignore the error here and return the stream even if we error?

Reply:
Makes sense, will ignore result.